lib/content-services/src/lib/content-node-share/content-node-share.dialog.spec.ts

```diff
@@ -342,5 +347,44 @@ describe('ShareDialogComponent', () => {
                 expiresAt: expiryDate
             });
         }));
+
+        it('should not update node when provided date is less than minDate', () => {
+            fixture.detectChanges();
+            clickExpireToggleButton();
+            fillInDatepickerInput('01.01.2010');
+
+            expect(component.form.invalid).toBeTrue();
+            expect(sharedLinksApiService.deleteSharedLink).not.toHaveBeenCalled();
+            expect(sharedLinksApiService.createSharedLinks).not.toHaveBeenCalled();
+        });
+
+        it('should not accept alphabets in the datepicker input', () => {
+            fixture.detectChanges();
+            clickExpireToggleButton();
+            fillInDatepickerInput('test');
+
+            expect(component.form.invalid).toBeTrue();
+            expect(component.form.controls['time'].value).toBeNull();
+        });
+
+        it('should show an error if provided date is invalid', () => {
+            fixture.detectChanges();
+            clickExpireToggleButton();
+            fillInDatepickerInput('32');
+            const error = fixture.debugElement.query(By.css('[data-automation-id="adf-share-link-input-warning"]'));
+
+            expect(error).toBeTruthy();
+            expect(component.time.hasError('invalidDate')).toBeTrue();
+        });
+
+        it('should show an error when provided date is valid', () => {
```

Comment:
incorrect title, should be: "should not show ..."

Reply:
corrected, ty